lib/helpers.js

```diff
@@ -11,6 +11,8 @@ const toBytes2 = s => toBytes(s, 2);
 const toBytes4 = s => toBytes(s, 4);
 const toBytes8 = s => toBytes(s, 8);
 
+const emptyBytes = size => toBytes('0', size);
+
```

Comment:
(nit) should we drop this if its not used anymore?

Reply:
removed